### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by bobmcwhirter**
- Add 'fractions' concept
- Add HOWTO for autodetecting and expliciting fractions.

-----

### CODE REVIEW

File: docs/concepts/fractions.adoc
Comment (Ladicek) [inline docs/concepts/fractions.adoc]: "to provided" -> "to provide"

"higher-order" sounds maybe too fancy?

"KeyCloak" -> "Keycloak" (not sure about RHSSO, maybe we should use the full form?)

File: docs/concepts/fractions.adoc
Comment (tradej) [inline docs/concepts/fractions.adoc]: Can you please explain this sentence? Does it mean that it does whatever you want it to?

File: docs/concepts/fractions.adoc
Comment (tradej) [reply docs/concepts/fractions.adoc]: Addressed in bobmcwhirter/wildfly-swarm#2.

**Review by tradej — CHANGES_REQUESTED** [review]
Nice job! For small details, see inline comments.

Please, also merge bobmcwhirter/wildfly-swarm#2.

**Comment by tradej** [comment]
Please do not merge until bobmcwhirter/wildfly-swarm#2 is re-merged--I have forgotten to push all my changes there.

**Review by tradej — APPROVED** [review]

**Comment by tradej** [comment]
Just in case: From my point of view, this can be merged.

**Pull request merged**
